modules/jwplayerTargeting.js

```diff
@@ -0,0 +1,158 @@
+import { config } from '../src/config.js';
+import { ajaxBuilder } from '../src/ajax.js';
+import { logError, isPlainObject } from '../src/utils.js';
+import { getGlobal } from '../src/prebidGlobal.js';
+import { module } from '../src/hook.js';
+
+const segCache = {};
```

Comment:
It seems weird that we have these global variables for the file that are getting updated and called by exported functions. Can you link to some doc on how this should be working? I think we should restructure functions and variables

Reply:
I think it's explained in the ticket in detail https://jwplayer.atlassian.net/browse/JW8-11235
Would you prefer if i summarized in the PR description ?

Reply:
PR description updated